bluepysnap/config.py

```diff
@@ -163,11 +164,54 @@ def to_dict(self):
 class CircuitConfig(Config):
     """Handle CircuitConfig."""
 
+    def __init__(self, *args):
+        """Initializes circuit config."""
+        super().__init__(*args)
+        self._populations = None
+        self._resolve_population_configs()
```

Comment:
Minor, if `_resolve_population_configs` returns a dict, it could be replaced with:
```python
self._populations = self._resolve_population_configs()
```
that would make clearer that `self._populations` is never None.

Reply:
Yeah, good call! I'll change it. I think I just tunnel-visioned myself out here.